CHANGELOG.rst

```diff
@@ -0,0 +1,15 @@
+================================
+Community.Proxysql Release Notes
+================================
+
+.. contents:: Topics
+
+
+v0.1.0
+======
+
+Release Summary
+---------------
+
+proxysql_* - all ProxySQL modules have been moved from ``community.general`` to the ``community.proxysql`` collection.  (https://github.com/ansible-collections/community.general/pull/624).
```

Comment:
This looks pretty horrible for a release summary :)

How about something like:
```.rst
All ProxySQL modules have been moved from ``community.general`` to the ``community.proxysql`` collection.

This changelog describes all changes made to the modules and plugins included in this collection since the migration to the ``community.proxysql`` collection. For additional changes between Ansible 2.9.0 and the first release of this collection, see the changelog of ``community.general``.
```

Reply:
😂 thanks for the honest review - you're totally right though, I'll update the fragment accordingly 🙂👍